Aleksandrova_Vilena/1/koans/about_blocks.rb

```diff
@@ -0,0 +1,96 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+class AboutBlocks < Neo::Koan
+  def method_with_block
+    result = yield
+    result
+  end
+
+  def test_methods_can_take_blocks
+    yielded_result = method_with_block { 1 + 2 }
+    assert_equal 3, yielded_result
+  end
+
+  def test_blocks_can_be_defined_with_do_end_too
+    yielded_result = method_with_block do 1 + 2 end
```

Comment:
```suggestion
    yielded_result = method_with_block do 
      1 + 2
    end
```
В руби do end блоки лучше разносить на разные строки :-)